app/javascript/utils/fieldSpecs.js

```diff
@@ -26,6 +24,7 @@ const dateIsJson = (date) => {
 }
 
 export const formatValue = (value, type) => {
+  console.log(value, type)
   if (type === 'date') {
```

Comment:
wondering why you added the currency format in the [expandable table](https://github.com/Exygy/sf-dahlia-lap/pull/321/files/d4c085152d5e9836f818b3abdea54d281f4e7bdd..0b943dbbf1c6a365a1cfb5a86215c6dd99fb90cf#diff-618b03bb9c05415bd74152c0747eb2cdR19) rather than here. Not a big deal. just curious because it looks like dates are formatted here.

Reply:
It looks like these are not formatted using this same fieldSpecs helpers, at least not the rental_assistances, they are built inline right before declaring the table and bypass the other formatting